vue/components/ui/organisms/table-menu/table-menu.stories.js

```diff
@@ -0,0 +1,158 @@
+                        Mars: "mars",
+                        Programmer: "programmer"
+                    },
+                    null
+                )
+            },
+            selectedIndex: {
+                type: Number,
+                default: number("Selected index", -1)
+            },
+            menuTitle: {
+                type: String,
+                default: text("Menu header title", "Arguments")
+            },
+            mockItems: {
+                default: () => [
+                    {
+                        id: 1,
+                        user: "Bill Gates",
+                        os: "Windows",
+                        alive: true,
+                        net: 123.23,
+                        programmer: true
+                    },
+                    {
+                        id: 2,
+                        user: "Steve Jobs",
+                        os: "Macintosh",
+                        alive: false,
+                        net: 123123,
+                        programmer: false
+                    },
+                    {
+                        id: 3,
+                        user: "Linus Torvalds",
+                        os: "Linux",
+                        alive: true,
+                        net: 321.123,
+                        programmer: true
+                    }
+                ]
+            },
+            mockColumns: {
+                type: Array,
+                default: () => [
+                    { value: "id", label: "ID", type: "number" },
+                    { value: "user", label: "User", type: "string" },
+                    { value: "os", label: "Operating System", type: "string" },
+                    { value: "alive", label: "Alive", type: "boolean" },
+                    { value: "programmer", label: "Programmer", type: "boolean" },
+                    { value: "net", label: "Net worth", type: "money", symbol: "$" }
+                ]
+            },
+            editColumns: {
+                type: Array,
+                default: ["user", "os", "alive", "programmer", "net"]
+            }
+        },
+        data: function() {
+            return {
+                reverseData: this.reverse,
+                sortData: this.sort,
+                menuVisibleData: this.menuVisible
+            };
+        },
+        watch: {
+            sort(value) {
+                this.sortData = value;
+            },
+            reverse(value) {
+                this.reverseData = value;
+            },
+            menuVisible(value) {
+                this.menuVisibleData = value;
+            }
+        },
+        template: `
+            <div>
+                <table-menu
+                    v-bind:columns="mockColumns"
+                    v-bind:items="mockItems"
+                    v-bind:sort.sync="sortData"
+                    v-bind:reverse.sync="reverseData"
+                    v-bind:menu-visible.sync="menuVisibleData"

```

Comment:
Are these sync working?

Shouldn't you have the events `update:sort`, `update:reverse` and `update:menuVisible` somewhere inside "table-menu.vue"?